### PR TITLE
Add cpanato as Emeritus Maintainers

### PR DESCRIPTION
**Pull request by carlisia**
Signed-off-by: Carlisia <carlisia@grokkingtech.io>

c/c and thank you @cpanato! ❤️

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by cpanato** [comment]
thanks for having me on the amazing project!! <3

**Review by dsu-igeek — APPROVED** [review]
LGTM

**Review by jenting — APPROVED** [review]
Thank you @cpanato

File: MAINTAINERS.md [hunk 1]
```diff
@@ -12,18 +12,19 @@
 | Bridget McErlean | [zubron](https://github.com/zubron) | [VMware](https://www.github.com/vmware/) |
 | Dave Smith-Uchida | [dsu-igeek](https://github.com/dsu-igeek) | [VMware](https://www.github.com/vmware/) |
 | JenTing Hsiao | [jenting](https://github.com/jenting) | [SUSE](https://github.com/SUSE/) 
-
+|
```
Comment (jenting) [inline MAINTAINERS.md:15]: This one is redundant.

**Review by jenting — APPROVED** [review]

**Review by ashish-amarnath — APPROVED** [review]

**Pull request merged**